holidays/countries/seychelles.py

```diff
@@ -0,0 +1,174 @@
+#  python-holidays
+#  ---------------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/dr-prodigy/python-holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.calendars.gregorian import JAN, MAR, MAY, SEP, OCT, DEC
+from holidays.groups import ChristianHolidays, InternationalHolidays, StaticHolidays
+from holidays.observed_holiday_base import ObservedHolidayBase, SUN_TO_NEXT_MON
+
+
+class Seychelles(ObservedHolidayBase, ChristianHolidays, InternationalHolidays):
+    """
+    https://www.psb.gov.sc/public-holidays
+    https://www.cbs.sc/PublicHolidays.html
+
+    [Act 19 of 1976, 1994 Amendment]
+        Oldest Seychelles Holidays Law available online in full.
+        https://seylii.org/akn/sc/act/1976/19/eng@2012-06-30
+    [Act 11 of 2014]
+        Holidays names changed.
+        https://seylii.org/akn/sc/act/2014/11/eng@2014-08-04
+    [Act 3 of 2017]
+        Added Easter Monday, repealing Liberation Day.
+        https://seylii.org/akn/sc/act/2017/3/eng@2017-04-12
+
+    Where any public holiday, except Sunday, falls on a Sunday the next following day,
+    not being itself a public holiday, shall be a public holiday.
+    """
+
+    country = "SC"
+    default_language = "en_SC"
+    # %s (observed).
+    observed_label = tr("%s (observed)")
+    supported_languages = ("en_SC", "en_US")
+
+    def __init__(self, *args, **kwargs):
+        ChristianHolidays.__init__(self)
+        InternationalHolidays.__init__(self)
+        StaticHolidays.__init__(self, SeychellesStaticHolidays)
+        kwargs.setdefault("observed_rule", SUN_TO_NEXT_MON)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # Earliest source is the 1994 amendment of Seychelles Public Holidays Act.
+        if self._year <= 1993:
+            return None
+
+        # New Year's Day.
+        self._add_new_years_day(tr("New Year's Day"))
+
+        # New Year Holiday.
+        self._add_observed(self._add_new_years_day_two(tr("New Year Holiday")))
+
+        # Good Friday.
+        self._add_observed(self._add_good_friday(tr("Good Friday")))
+
+        # Easter Saturday.
+        self._add_observed(self._add_holy_saturday(tr("Easter Saturday")))
+
+        if self._year >= 2017:
+            # Easter Monday.
+            self._add_observed(self._add_easter_monday(tr("Easter Monday")))
+
+        # Labour Day.
+        self._add_observed(self._add_labor_day(tr("Labour Day")))
+
+        # The Fete Dieu.
+        self._add_observed(self._add_corpus_christi_day(tr("The Fete Dieu")))
```

Comment:
```suggestion
        self._add_corpus_christi_day(tr("The Fete Dieu"))
```

Reply:
I haven't noticed until now that all Corpus Christi are always on Thursdays; good catch :eyes: